### PR TITLE
doc: add url Support in operation file Ref

### PR DESCRIPTION
**Pull request by shubham-cmyk**

### CODE REVIEW

**Comment by shubham-cmyk** [comment]
@eddycharly PTAL

File: website/docs/operations/apply.md [hunk 1]
```diff
@@ -27,6 +27,22 @@ Below is an example of using `apply` in a `Test` resource.
         # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/apply.md:30]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/apply.md [hunk 2]
```diff
@@ -68,6 +84,21 @@ Below is an example of using `apply` in a `TestStep` resource.
       # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/apply.md:87]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/assert.md [hunk 3]
```diff
@@ -36,6 +36,22 @@ Below is an example of using `assert` in a `Test` resource.
         # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/assert.md:39]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/assert.md [hunk 4]
```diff
@@ -77,6 +93,21 @@ Below is an example of using `assert` in a `TestStep` resource.
       # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/assert.md:96]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/create.md [hunk 5]
```diff
@@ -31,6 +31,21 @@ Below is an example of using `create` in a `Test` resource.
         # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/create.md:34]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/create.md [hunk 6]
```diff
@@ -72,6 +87,21 @@ Below is an example of using `create` in a `TestStep` resource.
       # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/create.md:90]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/error.md [hunk 7]
```diff
@@ -34,6 +34,22 @@ Below is an example of using `error` in a `Test` resource.
         # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/error.md:37]:
```suggestion
!!! example "Using an URL"
```

File: website/docs/operations/error.md [hunk 8]
```diff
@@ -75,6 +91,21 @@ Below is an example of using `error` in a `TestStep` resource.
       # ...
     ```
 
+!!! example "Using a URL"
```
Comment (eddycharly) [inline website/docs/operations/error.md:94]:
```suggestion
!!! example "Using an URL"
```

**Review by eddycharly — APPROVED** [review]

**Pull request merged**